lib/active_merchant/billing/gateway.rb

```diff
@@ -57,7 +57,7 @@ class Gateway
       include CreditCardFormatting
 
       DEBIT_CARDS = [ :switch, :solo ]
-      CURRENCIES_WITHOUT_FRACTIONS = %w(BIF BYR CLP CVE DJF GNF HUF ISK JPY KMF KRW PYG RWF UGX VND VUV XAF XOF XPF)
+      CURRENCIES_WITHOUT_FRACTIONS = %w(BIF BYR CLP CVE DJF GNF HUF IDR ISK JPY KMF KRW PYG RWF UGX VND VUV XAF XOF XPF)
```

Comment:
Hmm.  Stripe's list only has one currency that is not contained in `gateway`'s list. And I did not see `IDR` called out anywhere else. 

Currently, only Sagepay calls `gateway`'s list instead of their own.

In any case, it won't hurt to have a WorldPay-specific list, I'll go ahead and switch back to that and update.